web/src/styles/operator-shell.css

```
@@ -0,0 +1,2459 @@
+    56px 56px,
+    56px 56px;
+  color: var(--t-ink);
+  /* Interface is set in the UI sans; monospace is reserved for real data
+     (numbers, IDs, code, timestamps) via .opr-mono and the data classes below.
+     Terminal soul lives in the details, not a full-screen costume. */
+  font-family: var(--font-sans);
+  font-size: 14px;
+  line-height: 1.55;
+  overflow: hidden;
+  -webkit-font-smoothing: antialiased;
+  text-rendering: optimizeLegibility;
```

Stylelint (17.13.0)

[error] Expected "optimizeLegibility" to be "optimizelegibility" (value-keyword-case)

(value-keyword-case)